src/guide/introduction.md

```diff
@@ -13,7 +13,7 @@ footer: false
 
 <style src="@theme/styles/vue-mastery.css"></style>
 <div class="vue-mastery-link">
-  <a href="https://www.vuemastery.com/courses-path/beginner" target="_blank">
+  <a href="https://www.vuemastery.com/courses/" target="_blank">
```

Comment:
@tutorialcode could you please provide more context on why we're changing the URL? Beginner path URL is correct 🤔

Reply:
It was a dead link, so we re-routed it to the right URL internally as a quick fix. It should be the new URL.